site/docs/samples/notebook.md

```diff
@@ -44,23 +44,38 @@ Make a note of the service endpoint, bucket name, and access credentials. You wi
       export ACCESS_KEY="myaccesskey"
       export SECRET_KEY="mysecretkey"
       ```
-    2. Install Minio to the currently active namespace:
+    2. Install localstack to the currently active namespace:
       ```bash
-      kubectl create deployment minio --image=minio/minio:RELEASE.2021-02-14T04-01-33Z -- /bin/sh -ce "/usr/bin/docker-entrypoint.sh minio -S /etc/minio/certs/ server /export"
-      kubectl set env deployment/minio MINIO_ACCESS_KEY=${ACCESS_KEY} MINIO_SECRET_KEY=${SECRET_KEY}
-      kubectl wait --for=condition=available --timeout=120s deployment/minio
+      helm repo add localstack-charts https://localstack.github.io/helm-charts
+      helm install localstack localstack-charts/localstack --set startServices="s3" --set service.type=ClusterIP
       ```
-    3. Create a service to expose MinIO:
+    3. Create a container to use the AWS CLI and upload the dataset to it:
       ```bash
-      kubectl expose deployment minio --port 9000
+      cat << EOF | kubectl apply -f -
+      apiVersion: v1
```

Comment:
I think it might be too complex maybe? What do you think of instructions to download and use the cli instead?

Reply:
I changed the instructions to use AWS CLI...